appengine/flexible/tasks/create_app_engine_queue_task_test.py

```diff
@@ -18,7 +18,7 @@
 
 TEST_PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
 TEST_LOCATION = os.getenv("TEST_QUEUE_LOCATION", "us-central1")
-TEST_QUEUE_NAME = os.getenv("TEST_QUEUE_NAME", "my-appengine-queue")
+TEST_QUEUE_NAME = "my-static-appengine-queue"
```

Comment:
Queue has been restored. 

```suggestion
TEST_QUEUE_NAME = os.getenv("TEST_QUEUE_NAME", "my-appengine-queue")
```